README.md

```diff
@@ -41,114 +41,113 @@ cyrusauth:
 
 ## Installing ZNC
 
-If you are building from git, you will need to run `./autogen.sh` first to produce the `configure` script.
-Note that this requires `automake` and `gettext` to be installed.
+If you are building from git, you will need to run `./autogen.sh` first to
+produce the `configure` script. Note that this requires `automake` and
+`gettext` to be installed.
 
 Installation is done with the `./configure ; make ; make install` commands.
 
-You can use
-	./configure --help
-if you want to get a list of options, though the defaults should be suiting
-most needs. After you compiled it with make (or gmake if make doesn't work) you
-can install it with
-	make install
-though you don't need to as ZNC supports in-place execution.
+You can use `./configure --help` if you want to get a list of options,
+though the defaults should be suiting most needs. After you compiled it
+with make (or gmake if make doesn't work) you can install it with
+`make install` though you don't need to as ZNC supports in-place execution.
 
 ## Setting up znc.conf
 
 For setting up a configuration file in `~/.znc` you can simply do
-	znc --makeconf
-or
-	./znc --makeconf
-for in-place execution.
+`znc --makeconf` or `./znc --makeconf` for in-place execution.
 
-If you are using SSL you should do
-	znc --makepem
+If you are using SSL you should do `znc --makepem`
```

Comment:
It does.
--makepem is useful only if user has old config without znc.pem for any reason:
* User used --makeconf with ancient version, and didn't enable any SSL listener, but now wants to add one
* ZNC was compiled without SSL support
* The cert expired